tests/integration/cases/find_source_avaje/test.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2024 - 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I have two questions about this integration test:
- Should we add this test to the docker image test (using tag `macaron-docker-image`)?
- I notice that we only run the `find-source` commands in this test. Is this test case *only* intended to test whether the `find-source` commands exit with zero status code? If not, is it possible to validate the output files using the JSON schema file?

Reply:
- Probably makes sense to
- I think adding a validate step also makes sense